File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-person-language_item.ts


import { IGetPersonLanguage_Item } from './iget-person-language_item';

export class GetPersonLanguage_Item implements IGetPersonLanguage_Item {
  
	public PersonID: number = 0;
	public LanguageID: number = 0;
	public LanguageProficiencyID?: number;
	public LanguageDescription: string = "";
	public LanguageProficiencyCode: string = "";
	public LanguageProficiencyDescription: string = "";
	public ModifiedByAppUserID: number = 0;
	public LanguageCode: string = "";
  
}

File: GTTS/INL.TrainingService.Models/GetPersonLanguage_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
	public class GetPersonLanguage_Item : IGetPersonLanguage_Item
	{
		public long PersonID { get; set; }
		public int LanguageID { get; set; }
		public int? LanguageProficiencyID { get; set; }
		public string LanguageDescription { get; set; }
		public string LanguageProficiencyCode { get; set; }
		public string LanguageProficiencyDescription { get; set; }
		public int ModifiedByAppUserID { get; set; }
		public string LanguageCode { get; set; }
	}
}